packages/diagnostic/test/fixtures/app/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
should this inherit from the root base?

Reply:
no, want to keep test case independent

Reply:
ok we can have a root test tsconfig.json as well. eg. `tbconfig_test_base.json` and `"extends": "tbconfig_test_base.json"`

Reply:
Don't need for now.